app/controllers/concerns/helpers.rb

```diff
@@ -27,45 +27,35 @@ def convert_params_to_move(move, content, rowsize)
     ]
   end
 
-  def update_board_with_move(game, move)
-    game.mark_move_on_board(move)
-  end
-
-  def update_board_with_flag(game, move)
-    game.mark_flag_on_board(move)
-  end
-
   def bomb_positions_in_string(game_positions)
     game_positions.each_index.select { |i| game_positions[i].include? 'B' }
   end
 
   def find_revealed(cells_array)
-    cells_array.each_index.select{ |i| cells_array[i].status == 'revealed' }
+    cells_array.each_index.select{ |i| cells_array[i].status == 'revealed'}
   end
 
   def find_flags(cells_array)
     cells_array.each_index.select{ |i| cells_array[i].flag == 'F' }
   end
 
   def update_revealed_status(game, revealed_positions)
-    unless revealed_positions.nil? || revealed_positions.empty?
-      revealed = revealed_positions.split(',').map(&:to_i)
-      game.set_cell_status(revealed)
-    end
+    return if (revealed_positions.nil? || revealed_positions.empty?)
+    revealed = revealed_positions.split(',').map(&:to_i)
```

Comment:
`some_comma_separated_string`.split(',').map(*:to_i)` is reused several times. Is there a place you can extract and name this?